config-linux.md

```diff
@@ -330,14 +330,19 @@ The following parameters can be specified to set up the controller:
 
 * **`weight`** *(uint16, OPTIONAL)* - specifies per-cgroup weight. This is default weight of the group on all devices until and unless overridden by per-device rules.
 * **`leafWeight`** *(uint16, OPTIONAL)* - equivalents of `weight` for the purpose of deciding how much weight tasks in the given cgroup has while competing with the cgroup's child cgroups.
-* **`weightDevice`** *(array of objects, OPTIONAL)* - specifies the list of devices which will be bandwidth rate limited. The following parameters can be specified per-device:
+* **`weightDevice`** *(array of objects, OPTIONAL)* - specifies the list of devices which will be weight division of bandwidth. The following parameters can be specified per-device:
```

Comment:
> How about "specifies the list of devices which has its own specific weight and leafWeight"?

The “and” portion of this isn't necessarily true, since you can have just one or the other as well.  And this is less about *limiting* and more about *weighting*.  Also, [JSON uses “array” instead of ”list”][1].  So how about:

> * **`weightDevice`** *(array of objects, OPTIONAL)* - an array of per-device bandwidth weights.  Each entry has the following structure:

The each-entry line follows the current precedent:

```
$ git --no-pager grep -n 'entry.*following' v1.0.0-rc6
v1.0.0-rc6:config-linux.md:82:Each entry has the following structure:
v1.0.0-rc6:config-linux.md:115:Each entry has the following structure:
v1.0.0-rc6:config-linux.md:221:Each entry has the following structure:
v1.0.0-rc6:config-linux.md:394:Each entry has the following structure:
v1.0.0-rc6:config-linux.md:540:    Each entry has the following structure:
v1.0.0-rc6:config-linux.md:555:        Each entry has the following structure:
v1.0.0-rc6:config.md:165:    Each entry has the following structure:
```

[1]: http://json.org/